scripts/mkdocs_hooks.py

```
@@ -23,18 +23,23 @@ def resolve_relative_url(url, page, config):
+        doc_dir = posixpath.dirname(page.file.src_uri)
+
     # If the link starts with 'docs/', make it relative to the current page's depth
     if base_url.startswith('docs/'):
         target_path = base_url[5:]
-        doc_dir = posixpath.dirname(page.file.src_uri)
         if doc_dir:
             rel = os.path.relpath(target_path, doc_dir).replace('\\', '/')
         else:
             rel = target_path
         return rel + anchor
+    elif base_url.startswith('../../'):
```

Ruff (0.16.1)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)